src/init.c

```diff
@@ -560,8 +560,45 @@ static void hyper_flush_channel()
 	hyper_send_data_block(hyper_epoll.tty.fd, tty_buf->data, tty_buf->get);
 }
 
-void hyper_pod_destroyed(int failed)
+static void hyper_cleanup_mounts(struct hyper_pod *pod)
 {
+	int pid, pipe[2] = {-1, -1};
+	struct hyper_container *c;
+
+	if (pipe2(pipe, O_CLOEXEC) < 0) {
+		perror("create pipe for unmount failed");
+		return;
+	}
+
+	pid = fork();
+	if (pid < 0) {
+		perror("fork unmount process failed");
+		goto out;
+	} else if (pid == 0) {
+		if (hyper_enter_sandbox(pod, -1) < 0) {
+			hyper_send_type(pipe[1], -1);
+			_exit(-1);
+		}
```

Comment:
it seems the process doesn't need to enter the sanbox(pidns, utsns,ipcns), I can't find any pid,uts,ipc code in the following code.